ModularTegustation/tegu_items/lc13unique_items.dm

```diff
@@ -753,3 +753,117 @@
+
+/obj/item/lc13_shrimp_injector
+	name = "Shrimp Injector"
+	desc = "The injector contains a pink substance, is this really worth it? Usable by only clerks. Use in hand to activate. A small note on the injector states that 'clerk' means anyone with a job under service positions."
+	icon = 'ModularTegustation/Teguicons/teguitems.dmi'
+	icon_state = "oddity7_pink"
+
+/obj/item/lc13_shrimp_injector/attack_self(mob/living/carbon/human/user)
+	if(istype(user) && (user?.mind?.assigned_role in GLOB.service_positions))
+		if(user.faction == "shrimp")
+			to_chat(user,"<span class='warning'>The injector burns red before switching to green and dissapearing. You feel uneasy.</span>")
+			if(prob(70))
+				new /mob/living/simple_animal/hostile/shrimp(get_turf(user))
+			else
+				new /mob/living/simple_animal/hostile/shrimp_soldier(get_turf(user))
+			user.gib()
+		else
+			to_chat(user, "<span class='nicegreen'>The injector blinks green before it disintegrates. You feel pink? A catchy song about shrimp comes to mind.</span>")
+			user.faction |= "shrimp"
+			qdel(src)
+	else
+		to_chat(user, "<span class='notice'>The injector light flashes red. You aren't a clerk. Check the label before use.</span>")
+
+/obj/item/powered_gadget/clerkbot_gadget
+	name = "Instant Clerkbot Constructor"
+	desc = "An instant constructor for Clerkbots. Loyal little things that attack hostile creatures."
+	icon = 'ModularTegustation/Teguicons/teguitems.dmi'
+	icon_state = "gadget1"
+	default_icon = "gadget1" //roundabout way of making update item easily changed. Used in updateicon proc.
+	batterycost = 10000 //1 use before requires recharge
+
+/obj/item/powered_gadget/clerkbot_gadget/attack_self(mob/user)
+	..()
+	if(cell && cell.charge >= batterycost)
+		cell.charge = cell.charge - batterycost
+		icon_state = default_icon
+		if(istype(user) && (user?.mind?.assigned_role in GLOB.service_positions))
+			new /mob/living/simple_animal/hostile/clerkbot(get_turf(user))
+			to_chat(user, "<span class='nicegreen'>The Gadget turns warm and sparks.</span>")
+		else
+			to_chat(user, "<span class='notice'>The injector light flashes red. You aren't a clerk. Check the label before use.</span>")
+
+// Clerkbot Boio
+/mob/living/simple_animal/hostile/clerkbot
+	name = "A well rounded Clerkbot"
+	desc = "Trusted and loyal best friend."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "cube"
+	icon_living = "cube"
+	faction = list("neutral")
+	health = 150
+	maxHealth = 150
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	damage_coeff = list(RED_DAMAGE = 0.9, WHITE_DAMAGE = 0.9, BLACK_DAMAGE = 0.9, PALE_DAMAGE = 1.5)
+	melee_damage_lower = 12
+	melee_damage_upper = 14
+	robust_searching = TRUE
+	stat_attack = HARD_CRIT
+	del_on_death = TRUE
+	attack_verb_continuous = "buzzes"
+	attack_verb_simple = "buzz"
+	attack_sound = 'sound/weapons/bite.ogg'
+
+/mob/living/simple_animal/hostile/clerkbot/Initialize()
+	..()
+	addtimer(CALLBACK(src, .proc/die), 120 SECONDS)
```

Comment:
Use QDEL_IN instead.

Reply:
A what?

Reply:
```suggestion
	QDEL_IN(src, (120 SECONDS))
```

Reply:
Don't know why this was marked as resolved, it very clearly hasn't been resolved.